dashboard/test/ui/test_runner/test_runner_options_parser.rb

```diff
@@ -0,0 +1,210 @@
+require 'ostruct'
+require 'optparse'
+
+class TestRunnerOptionsParser
+  def initialize(argv)
+    @argv = argv
+    @options = OpenStruct.new
+    set_default_options
+  end
+
+  def parse
+    opt_parser = OptionParser.new do |opts|
+      configure_options(opts)
+    end
+
+    opt_parser.parse!(@argv)
+    standardize_feature_paths
+    set_db_access_options
+    if @options.config
+      @options.local = false
+    end
```

Comment:
I don't. The usage of those flags is the dark magic I am trying to solve or at least the data for.

I really don't think it is a good practice

Reply:
The only reason I am married to the method
standardize_feature_paths

is because it is so hard to undestand what this is doing, and while the method doesn't say much either, at least I undestand there is a pre processing for the features. 

  @options.features = @argv + (@options.features || []).map do |feature|
    feature.gsub(/^\.\//, '')